project/api/models/rationale.py

```diff
@@ -1,12 +1,10 @@
 from django.db import models
 
-from .bill import Bill
 from .representative import Representative
 from .vote import Vote
 
 
 class Rationale(models.Model):
-    bill = models.ForeignKey(Bill, default=None, null=True, on_delete=models.PROTECT)
```

Comment:
Remember to run `python manage.py makemigrations` to make the migrations related to model changes.